en/config/compass.md

```diff
@@ -77,6 +78,26 @@ Notes:
 - The amplitude and the speed of the partial rotations done in step 1 can affect the calibration quality.
   Following the advice above is usually enough.
 
+### Large Vehicle Calibration
+
+This calibration process leverages external knowledge of vehicle's orientation and location, and a World Magnetic Model (WMM) to calibrate the hard iron biases.
+
+1. Ensure GNSS Fix. This is required to find the expected Earth magnetic field in WMM tables.
+2. Align the vehicle to face True North.
+   Be as accurate as possible for best results.
+3. Open the [QGroundControl MAVLink Console](https://docs.qgroundcontrol.com/master/en/qgc-user-guide/analyze_view/mavlink_console.html) and send the following command:
+
+   ```sh
+   calibrate mag quick
```

Comment:
Just FMI, do you not have to specify `commander` when calling calibrate? https://docs.px4.io/main/en/modules/modules_system.html#usage-4

Reply:
Yes you do :|

Reply:
https://github.com/PX4/PX4-user_guide/pull/3546